src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Rectangle implements Figure {
+  shape: string = 'rectangle';
```

Comment:
TS will know that `shape` is `string` by first assign:
```suggestion
  shape = 'rectangle';
```